### DISCUSSION TITLE
Is Netmaker turning "Pro Only" regarding features ?

### DISCUSSION THREAD

**Discussion by gigadjo** [gravitl/netmaker#2824] [General]
Hello, and first of all, thanks for your work on Netmaker, it's a nice piece of software :-)
I've been using Nm for a few month, for the ease of use, compared to plain wireguard files :-)

I've recently updated to Netmaker 0.22.0, wich announced some exciting features.

When i updated to 0.22.0, i had a setup allowing me to : 

- give access to internet to some computers, that lived behind secured networks
- create a "private network" between my nas, ip cameras, recording device, so i can access my security cameras through a vm on my NAS when i'm away.
- create a 'more secure' network between two instances of homeassistant running on differents locations, allowing them to share a database server.

No big deal, but it was working ! And to avoid any problem with internet access on "iot" devices, i managed to add 0.0.0.0/0 to the egress ranges of the network they connect to, all ran smooth for month, and allowed me to "filter" their access using Squid/DNS Filtering  :-)

I updated to 0.22.0 because it was claming it could allow "internet access" on clients, via UI, with no needs to modify iptables rules, i thought "oh sweet !!" :-)

Updated following the doc, all went fine.. but.. the feature i was looking for, is "Pro ONLY" !!! When you want to add 0.0.0.0/0 to any ipv4 network, even if your NM server has got a public IP no filters, and your clients just needs to have those lines in their config (and the servers on his.. that's the problem here..).. it tells you to use "Remote Access Client" feature.. wich i searched, in vain : Pro ONLY as i figured out.

I understand the need for developpers to sell their products, and thought for a moment "Ok, let's see how much is it, i'd spend a few bucks than roll back a snapshot.." (i'm lazy, yeah..)

A very short moment, till i saw the price : 75$ per year.. ?! SaaS Only ?!
Oh, sure, the "Open" version is still there, but, without a feature it 'advertise' as 'new' and a 'Auto PRO installation' for newcomers.. :-)

I re-applied a snapshot, and went back on track.. but, to be honnest, i'll be looking around for a solution that doesn't try to "force" users into a paid (and expensive from my point of view) solution, by rebranding some features "Pro Only" :-)

I think it's ok to add new features over an existing system like Wireguard, be it a nice GUI, or even an easy way to manage inter-networking, and to sell it with support for corporate, but, to cut off some of the core features of a product like Wireguard (that has no problem routing packets to and from 0.0.0.0/0..) in order to sell them as "Remote Access Client", for paid only users, isn't a nice move imho.

I didn't went further, as the "Entreprise" version, seem out of scope, i'm just a "Home User".
Some kind of "HomeLab" liscence could be nice, designed for "home users" with some skills, that wants to keep "hand" on a critical solution such as network security, with a little ease from product like yours :-)

Best regards, keep the good work, 

DINET Joen

**Comment by Wombosvideo** [comment]
Having changed SSO to being premium only I can no longer log in to any of my accounts #2991.

As a software dev myself I understand the effort it takes to build great software, which Netmaker undoubtedly is, so I can also understand wanting something in return. But don't ever paywall previously free features without allowing previous users to continue using that feature for free. The only exception is a feature that is a deal-maker (as opposed to deal-breaker) which SSO, 2FA and other login or security related stuff definitely is not and should never be.

**Comment by gigadjo** [comment]
Hello, 

I Lately installed another instance of NM, for a friend needing to join two sites over VPN.. and found "Internet Gateway" to be back in the "Comunity Edition" (as of v0.25.0).

So i'd like to thank developers / contributors of Netmaker, for making it back to the "free tier" :+1: 

I have not tested the feature yet, but it allows to add 0.0.0.0/0 routes to the WG Clients config, while it was not possible on v0.24.x !
Great thanks ! A sign that this solution is not turning "Pro Only", thanks for the community, and those who don't need the full fledged set of features NM offers :-)

This is making NM a nice alternative, to some products like WG-EASY, that doesn't works that well compared to NM :1st_place_medal: 

Wish you the best :-)

/joen
